### ISSUE TITLE
Next.js tutorial not working (problems with webpack5)

### ISSUE THREAD

**Issue by IvanFLeon** [aws-amplify/docs#3327]
**Describe the bug**
When finishing the tutorial and publishing/hosting the app to amplify it completes correctly. But trying to access the website shows a 503 error.

**To Reproduce**
1. Go to https://docs.amplify.aws/start/getting-started/setup/q/integration/next
2. Follow the tutorial (A bit poorly written in my opinion, it doesn't mention adding Auth)
3. Host in amplify
4. Open generated link for your web app.
5. 503 message appears

**Expected behavior**
The web app should work.

The problem is fixed by opting out of webpack5 in the Next.js configuration.
Either way, the tutorial should mention that or there is a bug somewhere that should be fixed.
I imagine this is happening because of Next.js 11.

**Comment by StephenBlank** [comment]
I just went through the tutorial and was not able to reproduce the error using Next 11.0.1

Instead of adding auth and copying the Post schema from the docs, I used the default ToDo schema.

The `amplify add auth` step does need to be added. (Or, the step to copy custom schema needs to be removed.)

Also, the questions asked by the CLI during `amplify add api` have changed and need to be updated in the docs.

Never contributed before but willing to work on this if worthwhile
